app/views/categories/index.html.erb

```diff
@@ -0,0 +1,25 @@
+<section class="custom-container">
+  <%= render '/shared/nav_main',title: 'Spending Categories'  %>
+  <div class="custom-categories-container">
+    <% @categories.each do |category| %>
+    <%= link_to user_category_path(user_id:@user.id, id:category.id),
+        class:'custom-category-links' do %>
+        <div class="custom-category-card bg-primary-light text-primary">  
+          <div class="custom-category-card-wrapper">
+            <i class="material-icons outlined custom-material-icon"><%= category.icon%></i>
+            <h4 class="custom-category-card-title text-primary-light"><%= category.name%></h4>
+            <h4 class="custom-category-card-amount text-primary">$ <%= category.total_amount %></h4>
```

Comment:
- [OPTIONAL] Kindly follow the provided design. The card should have a category image, name, `created date`, and amount. kindly let display the creation date for each category card. 🛀🏻 

![Screenshot](https://github.com/admirerbrown/ROR-budget-App/assets/79058364/32290f91-a71b-48dd-b1af-9ee767564a3a)